xslt/Instrument.xsl

```diff
@@ -95,17 +95,22 @@ ome/ome.xsd: 979 # # This means that for more details on how this section of the
     <xsl:template match="LightSources">
         <xsl:for-each select="LightSource">
             <xsl:choose>
-                <xsl:when test="LightSourceType/Laser">
-                    <xsl:element name="ome:Laser">
-                        <xsl:attribute name="ID">
-                            <xsl:value-of select="@Id"/>
-                        </xsl:attribute>
+            <xsl:when test="LightSourceType/Laser">
+                <xsl:element name="ome:Laser">
+                    <xsl:attribute name="ID">
+                        <xsl:text>LightSource:</xsl:text>
```

Comment:
fwiw ... this general pattern applies to *all* of the ID fields in the schema.  See the respective fields in the OME schema and you'll see the rules for the respective fields

Reply:
@BrianWhitneyAI want to open a separate ticket to review our naming/usage to make sure this is satisfied everywhere?

Reply:
#23